rasa/core/training/interactive.py

```diff
@@ -1130,9 +1131,22 @@ async def _validate_action(
 
     Returns `True` if the prediction is correct, `False` otherwise."""
 
-    question = questionary.confirm(f"The bot wants to run '{action_name}', correct?")
-
-    is_correct = await _ask_questions(question, conversation_id, endpoint)
+    if action_name == ACTION_UNLIKELY_INTENT_NAME:
+        print(
+            f"The bot wants to run '{action_name}' "
+            f"(https://rasa.com/docs/rasa/default-actions#action_unlikely_intent) "
+            f"to indicate that this user message was unexpected "
```

Comment:
Should this be rephrased to `the last user message was unexpected`?

Reply:
I'd also explain what it means that a message is unexpected, e.g. `the last user message was unexpected at this point in the conversation`. How about asking somebody from CSE for review?

Reply:
The original message was written by @TyDunn so it's more like a question to you.

Reply:
`the last user message was unexpected at this point in the conversation` is an improvement. We discussed with CSE here: https://rasa-hq.slack.com/archives/C36SS4N8M/p1623402780458000